.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-    "editor.formatOnSave": true
+    "editor.formatOnSave": true,
+    "liveServer.settings.port": 5502
```

Comment:
what is this?

Reply:
I don't know. I didn't touch on this file.